com.unity.render-pipelines.core/ShaderLibrary/API/GLCore.hlsl

```diff
@@ -2,6 +2,14 @@
 #error GLES.hlsl should not be included if SHADER_API_GLCORE is not defined
 #endif
 
+// SHADER_AVAILABLE_XXX defines are not yet passed to compute shader atm
+// So we define it manually for compute atm.
+// It won't compile for devices that don't have cubemap array support but this is acceptable by now
+// TODO Remove this once SHADER_AVAILABLE_XXX are passed to compute shaders
+#ifdef SHADER_STAGE_COMPUTE
+#define SHADER_AVAILABLE_CUBEARRAY 1
+#endif
```

Comment:
Is it safe to consider any device with compute supports texture array? Because this is an include in core

Reply:
It's definitly a temporary workaround, we are discussing about this [here](https://unity.slack.com/archives/C02TGPN8MRV/p1644568756095959?thread_ts=1644242387.827459&cid=C02TGPN8MRV)

Reply:
Awaiting for some further [investigation](https://unity.slack.com/archives/C02TGPN8MRV/p1644853701512969?thread_ts=1644242387.827459&cid=C02TGPN8MRV) ⏳

Reply:
Switching back this PR to draft, we have to handle the `#pragma require cubearray` with care